stubs/Parsers/RulesParser.swift

```diff
@@ -31,18 +31,31 @@ enum JsonPathComponent {
 }
 
 enum Modification {
-    case add(dic: [String : Any])
+    case add(dic: [String: Any])
```

Comment:
Formatting change to stay consistent with the rest of the app.

Reply:
No `swiftlint` so I missed it 🤦‍♂️